### PR TITLE
Boardsource technik

### PR DESCRIPTION
**Pull request by daysgobye**
## Description
Added staggered version of technik keyboard the technik-s to the Boardsource folder.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by daysgobye** [comment]
This keyboard is 1 of 3 in a GB So I know for the next two is it better to keep it one keyboard per pr?

File: keyboards/boardsource/technik_s/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+![Technik-S](https://imgur.com/0QLgu5V)
+
+*A short description of the keyboard/project*
+
+* Keyboard Maintainer: [Boardsource](https://github.com/boardsource)
+* Hardware Supported: V1 pcb
+* Hardware Availability: [boardsource](https://boardsource.xyz/store/5ffb9b01edd0447f8023fdb2)
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/boardsource/technik_s/readme.md:11]: Could you add an example of how to enter the bootloader to flash?

Comment (daysgobye) [reply keyboards/boardsource/technik_s/readme.md:11]: Good call. I added a part about it.

**Review by drashna — APPROVED** [review]

**Comment by daysgobye** [comment]
are more changes needed?

File: keyboards/boardsource/technik_s/readme.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Technik-S
+
+![Technik-S](https://imgur.com/0QLgu5V)
```
Comment (fauxpark) [inline keyboards/boardsource/technik_s/readme.md:3]:
```suggestion
![Technik-S](https://i.imgur.com/0QLgu5V.jpeg)
```

File: keyboards/boardsource/technik_s/readme.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+*A short description of the keyboard/project*
```
Comment (fauxpark) [inline keyboards/boardsource/technik_s/readme.md:5]: Please fill this out.

File: keyboards/boardsource/technik_s/readme.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+Make example for this keyboard (after setting up your build environment):
+
+    make boardsource/technik_s:default
+
+Flashing example for this keyboard:
+
+    Reset keyboard by pushing reset button on back of pcb through access hole. Or hold down top left key while you plug in the usb cable.
+    Then run this command in your local qmk repo, or flash using qmk toolbox.
+    make boardsource/technik_s:default:flash
+
```
Comment (fauxpark) [inline keyboards/boardsource/technik_s/readme.md:20]:
```suggestion
    make boardsource/technik_s:default:flash

Reset keyboard by pushing reset button on back of pcb through access hole. Or hold down top left key while you plug in the usb cable.
Then run this command in your local qmk repo, or flash using qmk toolbox.
```

File: keyboards/boardsource/technik_s/readme.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
+
```
Comment (fauxpark) [inline keyboards/boardsource/technik_s/readme.md:24]:
```suggestion
```

File: keyboards/boardsource/technik_s/rules.mk [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+EXTRAFLAGS += -flto
```
Comment (fauxpark) [inline keyboards/boardsource/technik_s/rules.mk:25]:
```suggestion
LTO_ENABLE = yes
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**